model/project_ref.go

```diff
@@ -105,6 +105,8 @@ type ProjectRef struct {
 	// This is a temporary flag to enable individual projects to use repo settings
 	UseRepoSettings bool   `bson:"use_repo_settings" json:"use_repo_settings" yaml:"use_repo_settings"`
 	RepoRefId       string `bson:"repo_ref_id" json:"repo_ref_id" yaml:"repo_ref_id"`
+
+	TaskAnnotationSettings evergreen.AnnotationsSettings `bson:"task_annotation_settings,omitempty" bson:"task_annotation_settings,omitempty"`
```

Comment:
The unspoken flow of this section is that we only have it not in the first section if it needs a comment; in this case I think a comment is reasonable though (can you also move above 105; those are logically very different and I think make sense to keep at the end)